### PR TITLE
118 create test for get users

### PR DESCRIPTION
**Pull request by DLiSWE**

### CODE REVIEW

File: src/tests/auth.test.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ beforeEach(async () => {
 
 afterEach(async () => {
   await pgDataSource.destroy();
+  await new Promise<void>(resolve => setTimeout(() => resolve(), 500));
```
Comment (emir4ik-code) [inline src/tests/auth.test.ts:21]: What is this for?

File: src/tests/user.test.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const testCookie = authService.createTestCookie();
+beforeEach(async () => {
+  await pgDataSource.initialize();
+});
+
+afterEach(async () => {
+  await pgDataSource.destroy();
+});
+
+describe('Testing User Endpoints', () => {
```
Comment (ZohaibManzoor00) [inline src/tests/user.test.ts:20]: Maybe we can be more specific here and make the test more descriptive.  Maybe something like "Testing get users endpoint"

Comment (Matt-Tan15) [reply src/tests/user.test.ts:20]: I think it's fine, since we're building more than just get user endpoints

File: src/tests/auth.test.ts (hunk 1 above)
Comment (Matt-Tan15) [reply src/tests/auth.test.ts:21]: What does this do?

**Review by Matt-Tan15 — APPROVED** [review]

**Review by emir4ik-code — APPROVED** [review]

**Pull request merged**